app/controllers/game_move.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class GameMove
+  def all_moves(game, opponent, current_player, position, current_game)
+    if opponent == 'human'
+      game.move(position)
+      current_game.current_player = TicTacToeGame::HumanHumanToggle.new.current_turn(current_player)
+    else
+      if current_player == 'human'
+        game.move(position)
+      else
+        game.move(game.best_move)
+      end
+      current_game.current_player = TicTacToeGame::HumanComputerToggle.new.current_turn(current_player)
+
+    end
```

Comment:
Can you simplify the nested if statements here?

It looks like all the code branches here are mostly the same except for one branch that needs to pass in `game.best_move` to position and the other needs to simply pass in `position`.